src/EventPage.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react';
+import EventList from './componentsEventPage/EventList'
+import Header from './componentsEventPage/Header'
+import jsonEvents from './mockDB/events.json'
+
+export default function EventPage() {
+    const [categoryFilter, setCategoryFilter] = useState('all')
```

Comment:
You could consider having the state be a falsy state instead of `all` when there is no filter selected. That way, later when you filter your data, you can have the visible part of the frontend display anything you want if the state is falsy and you won't need to look for the specific value `all`

Reply:
I will check this, I have a button "all", which shows all lists, if nothing is filtered, so I thought this is also my default value.